webui/react/src/pages/ExperimentList.tsx

```diff
@@ -139,7 +139,7 @@ const ExperimentList: React.FC = () => {
       searchParams.append('user', URL_ALL);
     }
 
-    window.history.pushState(
+    window.history.replaceState(
```

Comment:
@apizzini the crux of this problem might be more related to the fact that table page offset is not stored in the URL. We intentionally made that choice of not including table page offset in the URL, So when the user clicks a bunch of different table pages, the URL is all the same, so when the user clicks the back button it just defaults to table page offset of 0 (first table page).

What do you think of the solution to add table page offset to the URL query params as well? That way with the pushing of routes, it'll store the page offset info properly and the back button should naturally behave as expected.